src/ConnectedMode/ConnectedModePackage.cs

```diff
@@ -64,6 +64,12 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             boundSolutionUpdateHandler = componentModel.GetService<BoundSolutionUpdateHandler>();
             timedUpdateHandler = componentModel.GetService<TimedUpdateHandler>();
 
+            // Trigger an initial update of suppressions (we might have missed the solution binding
+            // event from the ActiveSolutionBoundTracker)
+            // See https://github.com/SonarSource/sonarlint-visualstudio/issues/3886
+            var updater = componentModel.GetService<ISuppressionIssueStoreUpdater>();
+            updater.UpdateAllServerSuppressionsAsync().Forget();
```

Comment:
I've manually tested that this causes suppressions to be applied the first time a bound solution is open.
There is a slight delay as the issues are fetched from the server (i.e. the issue can appear in the Error List briefly).
However, this will always be the case whenever a bound solution is opened; it's not specific to startup.